{
  "Faults": [
    {
      "latitude": 50.89833208641456,
      "type": "PSI",
      "reference": "215616",
      "etaUpdate": true,
      "affectedCustomerCount": 100,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "932b4a7c-907c-423d-9dab-fc729f470efc",
      "minLongitude": -2.3571,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.3322498592628746,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-16T07:30:00.000+0000",
      "estimatedRestoration": "2023-04-22T15:30:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '215616'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/215616",
      "maxLongitude": -2.3084,
      "minLatitude": 50.8804,
      "name": "DT10 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "DT10 2AF",
        "DT10 2AG",
        "DT10 2AJ",
        "DT10 2AL",
        "DT10 2AN",
        "DT10 2AP",
        "DT10 2AQ",
        "DT10 2AW",
        "DT10 2DW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.3571,
              50.9148
            ],
            [
              -2.3084,
              50.9148
            ],
            [
              -2.3084,
              50.8804
            ],
            [
              -2.3571,
              50.8804
            ],
            [
              -2.3571,
              50.9148
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.9148,
      "etrUpdate": true
    },
    {
      "latitude": 51.624307467133335,
      "type": "LV",
      "reference": "KU4254",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-18T16:15:00.000+0000",
      "affectedCustomerCount": 29,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "61fad7fe-8b15-4558-96cc-0c527a1dfd3d",
      "minLongitude": -0.6642,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.6552369766810491,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T14:01:00.000+0000",
      "estimatedRestoration": "2023-04-19T02:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4254'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4254",
      "maxLongitude": -0.6477,
      "minLatitude": 51.6162,
      "name": "HP9 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "HP9 2DN",
        "HP9 2LS",
        "HP9 2RR",
        "HP9 2SR",
        "HP9 2TN",
        "HP9 2TP",
        "HP9 2TR",
        "HP9 2TS",
        "HP9 2TT",
        "HP9 2TU",
        "HP9 2TW",
        "HP9 2UD"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.6642,
              51.629
            ],
            [
              -0.6477,
              51.629
            ],
            [
              -0.6477,
              51.6162
            ],
            [
              -0.6642,
              51.6162
            ],
            [
              -0.6642,
              51.629
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-18T16:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.629,
      "etrUpdate": true
    },
    {
      "latitude": 51.39736338948108,
      "type": "LV",
      "reference": "KU4305",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-18T17:30:00.000+0000",
      "affectedCustomerCount": 12,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "a25e95d6-604c-4592-9d9d-867b0cb95d85",
      "minLongitude": -0.9188,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.9073482070550732,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T17:30:00.000+0000",
      "estimatedRestoration": "2023-04-19T05:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4305'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4305",
      "maxLongitude": -0.8963,
      "minLatitude": 51.3856,
      "name": "RG2 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG2 9HN",
        "RG2 9NX",
        "RG2 9PG",
        "RG2 9PH",
        "RG2 9PQ",
        "RG2 9PU",
        "RG2 9QA",
        "RG2 9QD",
        "RG2 9QE",
        "RG2 9QF"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.9188,
              51.4024
            ],
            [
              -0.8963,
              51.4024
            ],
            [
              -0.8963,
              51.3856
            ],
            [
              -0.9188,
              51.3856
            ],
            [
              -0.9188,
              51.4024
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-18T17:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.4024,
      "etrUpdate": true
    },
    {
      "latitude": 51.200433594375966,
      "type": "HV",
      "reference": "KU4460",
      "etaUpdate": true,
      "affectedCustomerCount": 105,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "25d7a748-65e5-4bcb-b41a-0780f87653cd",
      "minLongitude": -2.3546,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.305938042889437,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T20:25:00.000+0000",
      "estimatedRestoration": "2023-04-19T03:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4460'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4460",
      "maxLongitude": -2.2832,
      "minLatitude": 51.1736,
      "name": "BA11 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "BA11 2QS",
        "BA11 5EN",
        "BA11 5EQ",
        "BA11 5ES",
        "BA11 5EW",
        "BA11 5LG",
        "BA11 5LH",
        "BA11 5LJ",
        "BA11 5LL",
        "BA11 5LN",
        "BA11 5LP",
        "BA11 5LQ",
        "BA11 5LU",
        "BA11 5LW",
        "BA11 5LY",
        "BA11 5LZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.3546,
              51.2669
            ],
            [
              -2.2832,
              51.2669
            ],
            [
              -2.2832,
              51.1736
            ],
            [
              -2.3546,
              51.1736
            ],
            [
              -2.3546,
              51.2669
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-18T20:50:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.2669,
      "etrUpdate": true
    },
    {
      "latitude": 50.95669068166272,
      "type": "LV",
      "reference": "KU4475",
      "etaUpdate": true,
      "affectedCustomerCount": 24,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "2159146a-b830-4ded-8e58-42e415bd7644",
      "minLongitude": -1.2308,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.2278293702999403,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T22:04:00.000+0000",
      "estimatedRestoration": "2023-04-19T05:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4475'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4475",
      "maxLongitude": -1.2258,
      "minLatitude": 50.9555,
      "name": "SO32 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO32 1RN",
        "SO32 1RP",
        "SO32 1RQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.2308,
              50.9581
            ],
            [
              -1.2258,
              50.9581
            ],
            [
              -1.2258,
              50.9555
            ],
            [
              -1.2308,
              50.9555
            ],
            [
              -1.2308,
              50.9581
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.9581,
      "etrUpdate": true
    },
    {
      "latitude": 50.65989371583698,
      "type": "LV",
      "reference": "KU4491",
      "etaUpdate": true,
      "affectedCustomerCount": 9,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "831b15f7-6904-4829-91f5-b729635ff6df",
      "minLongitude": -1.1526,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.1512417303934723,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T22:51:00.000+0000",
      "estimatedRestoration": "2023-04-19T06:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4491'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4491",
      "maxLongitude": -1.1498,
      "minLatitude": 50.6563,
      "name": "PO36 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO36 8BW",
        "PO36 8DY",
        "PO36 8DZ",
        "PO36 8RB"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.1526,
              50.662
            ],
            [
              -1.1498,
              50.662
            ],
            [
              -1.1498,
              50.6563
            ],
            [
              -1.1526,
              50.6563
            ],
            [
              -1.1526,
              50.662
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-04-18T23:45:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.662,
      "etrUpdate": true
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T23:43:00.000+0000",
      "latitude": 51.578528528266446,
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4509'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4509",
      "reference": "KU4509",
      "maxLongitude": -0.5507,
      "etaUpdate": true,
      "minLatitude": 51.5752,
      "name": "SL9 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SL9 7DL",
        "SL9 7DP",
        "SL9 7DS",
        "SL9 7DT",
        "SL9 7DU",
        "SL9 7DW",
        "SL9 7DX",
        "SL9 7FD",
        "SL9 7JL",
        "SL9 7JT",
        "SL9 7JU",
        "SL9 7JX",
        "SL9 7JY",
        "SL9 7JZ",
        "SL9 7LA",
        "SL9 7LX",
        "SL9 7LZ",
        "SL9 7NG",
        "SL9 7NH",
        "SL9 7NJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.5663,
              51.582
            ],
            [
              -0.5507,
              51.582
            ],
            [
              -0.5507,
              51.5752
            ],
            [
              -0.5663,
              51.5752
            ],
            [
              -0.5663,
              51.582
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "426ca0f4-f617-44be-b11e-47836762a2b2",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.5663,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.582,
      "etrUpdate": true,
      "longitude": -0.5559834111251112,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T23:43:00.000+0000",
      "latitude": 51.63237035976497,
      "estimatedRestoration": "2023-04-19T02:45:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4517'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4517",
      "reference": "KU4517",
      "etaUpdate": true,
      "affectedCustomerCount": 26,
      "name": "HP8 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "HP8 4EN",
        "HP8 4EU",
        "HP8 4EW",
        "HP8 4EX",
        "HP8 4EY"
      ],
      "location": {
        "coordinates": [
          -0.5764392957942769,
          51.63237035976497
        ],
        "type": "Point"
      },
      "UUID": "c8b1b543-09d9-4029-be97-7d4ef0a07346",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "etrUpdate": true,
      "longitude": -0.5764392957942769,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T23:43:00.000+0000",
      "latitude": 51.60695516100467,
      "estimatedRestoration": "2023-04-19T02:45:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4555'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4555",
      "reference": "KU4555",
      "etaUpdate": true,
      "affectedCustomerCount": 29,
      "name": "UB9 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "UB9 6BE",
        "UB9 6BG",
        "UB9 6BH",
        "UB9 6BN",
        "UB9 6BQ",
        "UB9 6HF"
      ],
      "location": {
        "coordinates": [
          -0.49256638089653226,
          51.60695516100467
        ],
        "type": "Point"
      },
      "UUID": "8003968b-8aae-46f6-8632-c449b3c5294d",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "etrUpdate": true,
      "longitude": -0.49256638089653226,
      "resolved": false
    },
    {
      "latitude": 51.641694006685746,
      "type": "HV",
      "reference": "KU4594",
      "etaUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "7c287cdf-9001-41a8-bf90-30de0ed910f9",
      "minLongitude": -0.6168,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.5660164775829089,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T23:43:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4594'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4594",
      "maxLongitude": -0.5442,
      "minLatitude": 51.6208,
      "name": "HP7 & HP8 Areas",
      "otherUpdate": true,
      "affectedAreas": [
        "HP7 9NF",
        "HP7 9NG",
        "HP7 9PH",
        "HP7 9PN",
        "HP7 9PR",
        "HP7 9PS",
        "HP7 9PY",
        "HP7 9PZ",
        "HP7 9QA",
        "HP7 9QB",
        "HP7 9QD",
        "HP7 9QH",
        "HP7 9QJ",
        "HP7 9QL",
        "HP7 9QN",
        "HP7 9QQ",
        "HP7 9QT",
        "HP8 4AA",
        "HP8 4AB",
        "HP8 4AS",
        "HP8 4BH",
        "HP8 4BJ",
        "HP8 4BX",
        "HP8 4DF",
        "HP8 4DG",
        "HP8 4DY",
        "HP8 4EP",
        "HP8 4EQ",
        "HP8 4ER",
        "HP8 4ES",
        "HP8 4ET",
        "HP8 4EU",
        "HP8 4EW",
        "HP8 4EY",
        "HP8 4HA",
        "HP8 4HG",
        "HP8 4HH",
        "HP8 4HJ",
        "HP8 4HL",
        "HP8 4HN",
        "HP8 4HP",
        "HP8 4HQ",
        "HP8 4HR",
        "HP8 4HS",
        "HP8 4HT",
        "HP8 4HU",
        "HP8 4HW",
        "HP8 4HX",
        "HP8 4HY",
        "HP8 4HZ",
        "HP8 4JA",
        "HP8 4JB",
        "HP8 4JD",
        "HP8 4JF",
        "HP8 4JH",
        "HP8 4JJ",
        "HP8 4JL",
        "HP8 4JN",
        "HP8 4JR",
        "HP8 4JS",
        "HP8 4JT",
        "HP8 4JW",
        "HP8 4JX",
        "HP8 4LG",
        "HP8 4NB",
        "HP8 4ND",
        "HP8 4NE",
        "HP8 4NG",
        "HP8 4NH",
        "HP8 4NJ",
        "HP8 4NL",
        "HP8 4NN",
        "HP8 4NQ",
        "HP8 4NY",
        "HP8 4NZ",
        "HP8 4PG",
        "HP8 4PT",
        "HP8 4PX",
        "HP8 4PY",
        "HP8 4PZ",
        "HP8 4QA",
        "HP8 4QB",
        "HP8 4QD",
        "HP8 4QF",
        "HP8 4QG",
        "HP8 4QH",
        "HP8 4QJ",
        "HP8 4QL",
        "HP8 4QN",
        "HP8 4QP",
        "HP8 4QQ",
        "HP8 4QW",
        "HP8 4QY",
        "HP8 4QZ",
        "HP8 4RP",
        "HP8 4RT",
        "HP8 4RZ",
        "HP8 4SA",
        "HP8 4SB",
        "HP8 4SD",
        "HP8 4SF",
        "HP8 4SG",
        "HP8 4SH",
        "HP8 4SJ",
        "HP8 4SL",
        "HP8 4SN",
        "HP8 4SQ",
        "HP8 4SR",
        "HP8 4SS",
        "HP8 4SU",
        "HP8 4TH",
        "HP8 4TJ",
        "HP8 4TL",
        "HP8 4TU",
        "HP8 4TX",
        "HP8 4TY",
        "HP8 4TZ",
        "HP8 4UA",
        "HP8 4UB",
        "HP8 4UD",
        "HP8 4UJ",
        "HP8 4UT",
        "HP9 2RQ",
        "SL9 0PJ",
        "SL9 0RJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.6168,
              51.669
            ],
            [
              -0.5442,
              51.669
            ],
            [
              -0.5442,
              51.6208
            ],
            [
              -0.6168,
              51.6208
            ],
            [
              -0.6168,
              51.669
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "evChargersAffected": [
        "9e6640c5-6746-42b8-84a9-d27312a90cc3",
        "c03f5919-f858-47d5-ae5d-e0c73bf49f96",
        "336362a0-df3a-4b1f-b135-cf0bd08c2ccd",
        "2a34245c-ffd3-4737-8679-cf734f074191",
        "3b7f8152-46d2-4380-abf0-72f84c09028b",
        "20f960c6-82d9-401f-878a-ecef9b3c60c2",
        "6ec7fb94-d212-4df2-820a-c0dc7815e9c6"
      ],
      "maxLatitude": 51.669,
      "etrUpdate": true
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T23:43:00.000+0000",
      "latitude": 51.59963197880083,
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4601'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4601",
      "reference": "KU4601",
      "maxLongitude": -0.3393,
      "etaUpdate": true,
      "minLatitude": 51.5191,
      "name": "UB9 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "UB9 6EH",
        "UB9 6FJ",
        "UB9 6JX",
        "UB9 6JY",
        "UB9 6JZ",
        "UB9 6LA",
        "UB9 6LB",
        "UB9 6LJ",
        "UB9 6LP",
        "UB9 6LW",
        "W7 3QX"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.3437,
              51.5199
            ],
            [
              -0.3393,
              51.5199
            ],
            [
              -0.3393,
              51.5191
            ],
            [
              -0.3437,
              51.5191
            ],
            [
              -0.3437,
              51.5199
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "234cac2b-fb44-47bf-8c61-88a8e34562c2",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.3437,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.5199,
      "etrUpdate": true,
      "longitude": -0.46590935130362643,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T23:43:00.000+0000",
      "latitude": 51.60240291723658,
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4603'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4603",
      "reference": "KU4603",
      "etaUpdate": true,
      "name": "UB8 & UB9 Areas",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "UB8 1PS",
        "UB9 6EU",
        "UB9 6EX",
        "UB9 6EY",
        "UB9 6EZ",
        "UB9 6HA",
        "UB9 6HB",
        "UB9 6HD",
        "UB9 6HX",
        "UB9 6PR",
        "UB9 6PS",
        "UB9 6TQ"
      ],
      "location": {
        "coordinates": [
          -0.47637140035954695,
          51.60240291723658
        ],
        "type": "Point"
      },
      "UUID": "774b8792-11a2-4f91-96ca-edc06778eeb6",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "etrUpdate": true,
      "longitude": -0.47637140035954695,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T23:43:00.000+0000",
      "latitude": 51.628530290638416,
      "estimatedRestoration": "2023-04-19T02:45:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4606'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4606",
      "reference": "KU4606",
      "etaUpdate": true,
      "affectedCustomerCount": 15,
      "name": "HP8 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "HP8 4DF",
        "HP8 4DG",
        "HP8 4DH",
        "HP8 4DQ",
        "HP8 4DY"
      ],
      "location": {
        "coordinates": [
          -0.5771374025462599,
          51.628530290638416
        ],
        "type": "Point"
      },
      "UUID": "e0b1669c-31ee-4643-ad5b-3bb5c4a8b8a9",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "etrUpdate": true,
      "longitude": -0.5771374025462599,
      "resolved": false
    },
    {
      "latitude": 51.552754493542366,
      "type": "LV",
      "reference": "KU4618",
      "etaUpdate": true,
      "affectedCustomerCount": 16,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "14b497e7-1ab1-453e-8043-7a747be281a2",
      "minLongitude": -0.4956,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.4914305420376634,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T23:43:00.000+0000",
      "estimatedRestoration": "2023-04-19T02:45:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4618'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4618",
      "maxLongitude": -0.489,
      "minLatitude": 51.5495,
      "name": "UB9 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "UB9 4BA",
        "UB9 4BD",
        "UB9 4BE"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.4956,
              51.5541
            ],
            [
              -0.489,
              51.5541
            ],
            [
              -0.489,
              51.5495
            ],
            [
              -0.4956,
              51.5495
            ],
            [
              -0.4956,
              51.5541
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5541,
      "etrUpdate": true
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T23:43:00.000+0000",
      "latitude": 51.59670186129516,
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4685'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4685",
      "reference": "KU4685",
      "maxLongitude": -0.5514,
      "etaUpdate": true,
      "minLatitude": 51.5918,
      "name": "SL9 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SL9 8JD",
        "SL9 8JE",
        "SL9 8JF",
        "SL9 8JG",
        "SL9 8JS",
        "SL9 8JW",
        "SL9 8NB",
        "SL9 8NJ",
        "SL9 8NL",
        "SL9 8NN",
        "SL9 8NP",
        "SL9 8NR",
        "SL9 8NT",
        "SL9 8NU",
        "SL9 8NW",
        "SL9 8NX",
        "SL9 8NZ",
        "SL9 8PD",
        "SL9 8PZ",
        "SL9 8QA",
        "SL9 8QB",
        "SL9 8SF",
        "SL9 8SG",
        "SL9 8UD"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.562,
              51.6012
            ],
            [
              -0.5514,
              51.6012
            ],
            [
              -0.5514,
              51.5918
            ],
            [
              -0.562,
              51.5918
            ],
            [
              -0.562,
              51.6012
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "0115fec8-8fae-4d28-8306-45a97aa18b3e",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.562,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.6012,
      "etrUpdate": true,
      "longitude": -0.5560279863406116,
      "resolved": false
    },
    {
      "latitude": 51.4689547098913,
      "type": "LV",
      "reference": "KU4508",
      "etaUpdate": true,
      "affectedCustomerCount": 15,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "60cfe8ff-f5fc-480f-afcb-7aaf32baf6a6",
      "minLongitude": -0.3751,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.36130610851296174,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T23:42:00.000+0000",
      "estimatedRestoration": "2023-04-19T02:45:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4508'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4508",
      "maxLongitude": -0.3572,
      "minLatitude": 51.4626,
      "name": "TW3 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "TW3 1DA",
        "TW3 1DE",
        "TW3 1HE",
        "TW3 1HL",
        "TW3 1QT"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.3751,
              51.4705
            ],
            [
              -0.3572,
              51.4705
            ],
            [
              -0.3572,
              51.4626
            ],
            [
              -0.3751,
              51.4626
            ],
            [
              -0.3751,
              51.4705
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.4705,
      "etrUpdate": true
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T23:42:00.000+0000",
      "latitude": 51.6269821392854,
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4544'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4544",
      "reference": "KU4544",
      "etaUpdate": true,
      "name": "HP8 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "HP8 4DA",
        "HP8 4DB",
        "HP8 4DD",
        "HP8 4DE",
        "HP8 4DF",
        "HP8 4DL",
        "HP8 4DP",
        "HP8 4DR",
        "HP8 4DS",
        "HP8 4DT",
        "HP8 4DU",
        "HP8 4DW"
      ],
      "location": {
        "coordinates": [
          -0.5792373559521079,
          51.6269821392854
        ],
        "type": "Point"
      },
      "UUID": "4fb521f4-a923-42fb-b51e-ba710ae18546",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "etrUpdate": true,
      "longitude": -0.5792373559521079,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T23:42:00.000+0000",
      "latitude": 51.5641380921046,
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4510'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4510",
      "reference": "KU4510",
      "maxLongitude": -0.4858,
      "etaUpdate": true,
      "minLatitude": 51.5513,
      "name": "UB9 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "UB9 4DF",
        "UB9 4DQ",
        "UB9 4EA",
        "UB9 4EB",
        "UB9 4EF",
        "UB9 5AT",
        "UB9 5AU",
        "UB9 5AW",
        "UB9 5BP",
        "UB9 5BU",
        "UB9 5BX",
        "UB9 5DB"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.5055,
              51.5693
            ],
            [
              -0.4858,
              51.5693
            ],
            [
              -0.4858,
              51.5513
            ],
            [
              -0.5055,
              51.5513
            ],
            [
              -0.5055,
              51.5693
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "6b24fb4f-c7c9-4f1c-93a7-b87b337a56b9",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.5055,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.5693,
      "etrUpdate": true,
      "longitude": -0.49630548762940746,
      "resolved": false
    },
    {
      "latitude": 51.62038054236624,
      "type": "LV",
      "reference": "KU4520",
      "etaUpdate": true,
      "affectedCustomerCount": 28,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "8ef8a3e5-6ce1-4aee-a000-de6411e6e2cc",
      "minLongitude": -0.5542,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.553340914628296,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T23:42:00.000+0000",
      "estimatedRestoration": "2023-04-19T02:45:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4520'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4520",
      "maxLongitude": -0.5482,
      "minLatitude": 51.6171,
      "name": "HP8 & SL9 Areas",
      "otherUpdate": true,
      "affectedAreas": [
        "HP8 4QA",
        "SL9 0HX",
        "SL9 0HY",
        "SL9 0HZ",
        "SL9 0JL",
        "SL9 0LG",
        "SL9 0LH",
        "SL9 0LL",
        "SL9 0LQ",
        "SL9 0LR"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.5542,
              51.6215
            ],
            [
              -0.5482,
              51.6215
            ],
            [
              -0.5482,
              51.6171
            ],
            [
              -0.5542,
              51.6171
            ],
            [
              -0.5542,
              51.6215
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.6215,
      "etrUpdate": true
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T23:42:00.000+0000",
      "latitude": 51.61223918206182,
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4543'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4543",
      "reference": "KU4543",
      "maxLongitude": -0.6428,
      "etaUpdate": true,
      "minLatitude": 51.6101,
      "name": "HP9 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "HP9 1BY",
        "HP9 1HQ",
        "HP9 1NL",
        "HP9 1NN",
        "HP9 1QB",
        "HP9 1QD",
        "HP9 1QU",
        "HP9 2BW",
        "HP9 2FL",
        "HP9 2FY",
        "HP9 2LZ",
        "HP9 2NA",
        "HP9 2NB",
        "HP9 2ND",
        "HP9 2NE",
        "HP9 2NF",
        "HP9 2NJ",
        "HP9 2PD",
        "HP9 2PE",
        "HP9 2PL",
        "HP9 2PP",
        "HP9 2QB"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.6539,
              51.615
            ],
            [
              -0.6428,
              51.615
            ],
            [
              -0.6428,
              51.6101
            ],
            [
              -0.6539,
              51.6101
            ],
            [
              -0.6539,
              51.615
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "f1d4d4b4-3029-4df5-9511-89b48f113d1f",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.6539,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.615,
      "etrUpdate": true,
      "longitude": -0.6473035673098929,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T22:18:00.000+0000",
      "latitude": 51.20670490941269,
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4566'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4566",
      "reference": "KU4566",
      "maxLongitude": -2.2141,
      "etaUpdate": true,
      "minLatitude": 51.0896,
      "name": "BA12 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BA11 2E",
        "BA11 2QS",
        "BA11 5AL",
        "BA11 5EN",
        "BA11 5EQ",
        "BA11 5ES",
        "BA11 5EW",
        "BA11 5JG",
        "BA11 5JL",
        "BA11 5JN",
        "BA11 5JQ",
        "BA11 5JR",
        "BA11 5JS",
        "BA11 5JT",
        "BA11 5JW",
        "BA11 5LA",
        "BA11 5LD",
        "BA11 5LE",
        "BA11 5LF",
        "BA11 5LG",
        "BA11 5LH",
        "BA11 5LJ",
        "BA11 5LL",
        "BA11 5LN",
        "BA11 5LP",
        "BA11 5LQ",
        "BA11 5LR",
        "BA11 5LU",
        "BA11 5LW",
        "BA11 5LY",
        "BA11 5LZ",
        "BA12",
        "BA12 6JB",
        "BA12 7ER",
        "BA12 7FD",
        "BA12 7NE",
        "BA12 7NF",
        "BA12 7NJ",
        "BA12 7NL",
        "BA12 7NN",
        "BA12 7NP",
        "BA12 7NQ",
        "BA12 7NR",
        "BA12 7NS",
        "BA12 7NT",
        "BA12 7NW",
        "BA12 7NX",
        "BA12 7PA",
        "BA12 7PB",
        "BA12 7PD",
        "BA12 7PE",
        "BA12 7PF",
        "BA12 7PG",
        "BA12 7PH",
        "BA12 7PJ",
        "BA12 7PL",
        "BA12 7PN",
        "BA12 7PP",
        "BA12 7PQ",
        "BA12 7PR",
        "BA12 7PS",
        "BA12 7PT",
        "BA12 7PW",
        "BA12 7QD",
        "BA12 7QE",
        "BA12 7QF",
        "BA12 7QG",
        "BA12 7QH",
        "BA12 7QL",
        "BA12 7QQ",
        "BA12 7QZ",
        "BA13 4AU"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.3546,
              51.2669
            ],
            [
              -2.2141,
              51.2669
            ],
            [
              -2.2141,
              51.0896
            ],
            [
              -2.3546,
              51.0896
            ],
            [
              -2.3546,
              51.2669
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "74e08c9c-e5c6-41d3-93c4-7e8838a244d7",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -2.3546,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.2669,
      "etrUpdate": true,
      "longitude": -2.2791971818328953,
      "resolved": false
    },
    {
      "latitude": 56.46841414351642,
      "type": "LV",
      "reference": "KU4470",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-04-18T22:15:00.000+0000",
      "affectedCustomerCount": 10,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "aebc1aeb-1617-4cfd-bc53-67f14252d69f",
      "minLongitude": -2.9559,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.9405567088640217,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T21:47:00.000+0000",
      "estimatedRestoration": "2023-04-19T01:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4470'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4470",
      "maxLongitude": -2.9317,
      "minLatitude": 56.4646,
      "name": "DD4 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "DD1 3JX",
        "DD4 6BD",
        "DD4 6LA",
        "DD4 6LB",
        "DD4 6LD",
        "DD4 7FE",
        "DD4 7JE",
        "DD4 7JP",
        "DD4 7JR",
        "DD4 7JU"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.9559,
              56.4724
            ],
            [
              -2.9317,
              56.4724
            ],
            [
              -2.9317,
              56.4646
            ],
            [
              -2.9559,
              56.4646
            ],
            [
              -2.9559,
              56.4724
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 56.4724,
      "etrUpdate": true
    },
    {
      "latitude": 57.479054866429564,
      "type": "LV",
      "reference": "KU4474",
      "etaUpdate": true,
      "affectedCustomerCount": 6,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "82f28393-888d-4e6d-8400-5f6ed4c2aa23",
      "minLongitude": -4.2299,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -4.2275825430911675,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-04-18T22:46:00.000+0000",
      "estimatedRestoration": "2023-04-19T05:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KU4474'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KU4474",
      "maxLongitude": -4.226,
      "minLatitude": 57.4771,
      "name": "IV1 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "IV1 1DW",
        "IV1 1EH",
        "IV1 1EY",
        "IV1 1QU",
        "IV1 1QY"
      ],
      "location": {
        "coordinates": [
          [
            [
              -4.2299,
              57.4804
            ],
            [
              -4.226,
              57.4804
            ],
            [
              -4.226,
              57.4771
            ],
            [
              -4.2299,
              57.4771
            ],
            [
              -4.2299,
              57.4804
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 57.4804,
      "etrUpdate": true
    }
  ]
}
